pkg/distro/image_config_test.go

```diff
@@ -166,3 +166,71 @@ func TestImageConfigInheritFrom(t *testing.T) {
 		})
 	}
 }
+
+func TestImageConfiDNCConfigNone(t *testing.T) {
```

Comment:
```suggestion
func TestImageConfigDNFConfigNone(t *testing.T) {
```
AI found a double-typo here. :)